File: 0009-palindrome-number/0009-palindrome-number.cs
public class Solution {
    public bool IsPalindrome(int x) {
        string myY = "";
        string myX = x.ToString();
        
        for (int i = myX.Length - 1; i >= 0; --i) {
            myY = myY + myX[i];
        }
        
        return myY == myX;
    }
}

File: 0009-palindrome-number/0009-palindrome-number.java
class Solution {
    public boolean isPalindrome(int x) {
        String myY = "";
        String myX = Integer.toString(x);

        for (int i = myX.length() - 1; i >= 0; --i) {
            myY = myY + myX.charAt(i);
        }

        return myY.equals(myX);
    }
}